src/azure-cli/azure/cli/command_modules/appconfig/custom.py

```diff
@@ -150,9 +161,21 @@ def update_configstore(cmd,
 
         update_params.encryption = EncryptionProperties(key_vault_properties=key_vault_properties)
 
-    return client.begin_update(resource_group_name=resource_group_name,
-                               config_store_name=name,
-                               config_store_update_parameters=update_params)
+    try:
+        return client.begin_update(
+            resource_group_name=resource_group_name,
+            config_store_name=name,
+            config_store_update_parameters=update_params,
+        )
+    # Incase premium sku is not available in a given region yet. Temporary fix will be removed when premium is in all regions
+    except HttpResponseError as exception:
+        if exception.status_code == StatusCodes.BAD_REQUEST:
```

Comment:
Can all this be 1 `if` statement so we would not need multiple `else` statements doing the same thing?